types/geojson/index.d.ts

```diff
@@ -60,7 +60,10 @@ export interface GeoJsonObject {
 /**
  * Union of GeoJSON objects.
  */
-export type GeoJSON = Geometry | Feature | FeatureCollection;
+export type GeoJSON<G extends Geometry | null = Geometry | null, P extends GeoJsonProperties = GeoJsonProperties> =
+    | Geometry
```

Comment:
Would it make more sense for this to be `G`? Or at least `Exclude<G, null>`?

This feels a little funny to allow:

```ts
const geojson: GeoJSON<Point> = lineString;
```

Reply:
You are entirely correct! I will make this change